cypress/page/iframe.page.ts

```diff
@@ -0,0 +1,28 @@
+class IFramePage {
+    private iFramePageURL: string;
+    private iFrame: string;
+    private iFrameTitle: string;
+    private cssTitle: string;
+
+    constructor() {
+        this.iFramePageURL = "https://www.w3schools.com/html/html_iframe.asp";
+        this.iFrameTitle = "#main h1";
+        this.cssTitle = "#topnav a[title='CSS Tutorial']";
+        this.iFrame = "[title='W3Schools HTML Tutorial']"
+    }
+
+    public visit(){
+       cy.visit(this.iFramePageURL);
+      }
+      
+    public  getFrameTitle(){
+        return cy.iframe(this.iFrame).find(this.iFrameTitle);
+      }
+      
+    public goToCssPageInFrame(){
+        cy.iframe(this.iFrame).find(this.cssTitle).click({force: true});
+        cy.wait(2000)
```

Comment:
instead of the wait, you can use the frameLoaded command:
`cy.frameLoaded(this.iFrame, {url: urlToCssPage});`